lightning/pkg/importer/precheck_impl.go

```diff
@@ -576,32 +576,43 @@ func (ci *localTempKVDirCheckItem) Check(ctx context.Context) (*precheck.CheckRe
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	localAvailable := int64(storageSize.Available)
+	localAvailable := int64(storageSize.Available)           //Available disk space in bytes
+	availableStr := units.BytesSize(float64(localAvailable)) //Human-readable availabe space
```

Comment:
```suggestion
	localAvailable := int64(storageSize.Available)           // Available disk space in bytes
	availableStr := units.BytesSize(float64(localAvailable))
```
I think we don't need to add comment for every variable